src/main/java/com/wanikani/api/v2/request/BaseRequest.java

```diff
@@ -0,0 +1,35 @@
+package com.wanikani.api.v2.request;
+
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+
+import java.time.Instant;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+
+@SuperBuilder(toBuilder = true)
+@ToString
+public abstract class BaseRequest implements Request {
```

Comment:
Right now both could be. CollectionRequest can't for the paging change I'm working on because I need to generically identify "something that can have pageAfterId set on it", and that class fills that role.